devkit/samples/attachments/attachments.feature

```diff
@@ -15,30 +15,26 @@ Feature: Attachments
 
     When the string "hello" is attached as "application/octet-stream"
 
+  Scenario: Log text
+    When the string "hello" is logged
+
+  Scenario: Log ANSI coloured text
+    When text with ANSI escapes is logged
+
   Scenario: Log JSON
      When the following string is attached as "application/json":
        ```
        {"message": "The <b>big</b> question", "foo": "bar"}
        ```
 
-  Scenario: Log text
-    When the string "hello" is logged
-
-  Scenario: Log ANSI coloured text
-     When text with ANSI escapes is logged
-
   Scenario: Byte arrays are base64-encoded regardless of media type
     When an array with 10 bytes is attached as "text/plain"
 
-  Scenario: Streams are always base64-encoded
+  Scenario: Attaching JPEG images
     When a JPEG image is attached
 
-  Scenario: Attaching images in examples
```

Comment:
Why have we removed this one?

Reply:
We've just simplified the code. Examples tables are covered in a diff feature